analyzer/src/funTest/kotlin/NpmTest.kt

```diff
@@ -114,24 +114,6 @@ class NpmTest : WordSpec() {
                 yamlMapper.writeValueAsString(result) shouldBe expectedResult
             }
 
-            "show error if multiple lockfiles are present" {
```

Comment:
This test should be replaced with one that verifies the new behaviour, currently "if NPM and Yarn lockfiles are present Yarn is used". But also see my other comment.